frontend/src/components/Chat.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FunctionComponent, useState, useEffect } from "react";
+
+const styles = {
+  chatBox: {
+    border: "10px solid white",
+    borderRadius: "20%",
+    color: "white",
+    maxWidth: "10vw",
+    height: "50vh"
+  }
+};
+
+const display = {
+  messages: []
+};
+
+type MessageProps = {
+  message: string;
+};
+
+// eslint-disable-next-line react/prop-types
```

Comment:
If you declare the type in the function parameter you won't need the disable the rule.